### ISSUE TITLE
[TalkBack] No Feedback when Dragging items on Nexus 7 Home Screens

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#184]
```
What steps will reproduce the problem?
1. On the Nexus 7, go to the default home screen.
2. Select any icon and double tap and hold to move it to a new location
3. Move your finger around the screen.

What is the expected output? What do you see instead?
Feedback should be given as the finger is moved to indicate where the icon will 
be dropped  (on top of another app, the trash can, dock, etc.)
Currently, no feedback is given, so there is no way to accessibly move icons 
around the home screen.

What version of the product are you using? On what operating system?
Android 4.2.2 on a Nexus 7, stock firmware



```

Original issue reported on code.google.com by `j...@bestmidi.com` on 23 Feb 2013 at 8:17

**Comment by GoogleCodeExporter** [comment]
```
Cannot fix from TalkBack, feature request filed against Android.
```

Original comment by `alanv@google.com` on 6 Mar 2013 at 12:34
- Changed title: **[TalkBack] No Feedback when Dragging items on Nexus 7 Home Screens**
- Changed state: **WontFix**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**
